doc/topics/releases/fluorine.rst

```diff
@@ -80,6 +80,387 @@ orchestration state runner.
+:mod:`configuration cleanup <salt.modules.iosconfig.clean>`,
+:mod:`tree representation of the config <salt.modules.iosconfig.tree>`, etc.
+
+NAPALM
+------
+
+Commit At and Commit Confirmed
+""""""""""""""""""""""""""""""
+
+Beginning with this release, NAPALM users are able to execute scheduled commits
+(broadly known as "commit at") and "commit confirmed" (revert the configuration 
+change unless the user confirms by running another command). These features are 
+available via the ``commit_in``, ``commit_at``, ``revert_in``, or ``revert_at``
+arguments for the
+:mod:`net.load_config <salt.modules.napalm_network.load_config>` and
+:mod:`net.load_template <salt.modules.napalm_network.load_template>` execution 
+functions, or :mod:`netconfig.managed <salt.states.netconfig.managed>`.
+
+The counterpart execution functions
+:mod:`net.confirm_commit <salt.modules.napalm_network.confirm_commit>`, or
+:mod:`net.cancel_commit <salt.modules.napalm_network.cancel_commit>`, as well 
+as the State functions
+:mod:`netconfig.commit_cancelled <salt.states.netconfig.commit_cancelled>`, or
+:mod:`netconfig.commit_confirmed <salt.states.netconfig.commit_confirmed>` can 
+be used to confirm or cancel a commit.
+
+Please note that the commit confirmed and commit cancelled functionalities are
+available for any platform whether the network devices supports the features 
+natively or not. However, be cautious and make sure you read an understood the 
+caveats before using them in production.
+
+Multiple Templates Rendered Simultaneously
+""""""""""""""""""""""""""""""""""""""""""
+
+The ``template_name`` argument of the
+:mod:`net.load_template <salt.modules.napalm_network.load_template>` Execution 
+and :mod:`netconfig.managed <salt.states.netconfig.managed>` State function now
+supports a list of templates. This is particularly useful when a very large 
+Jinja template is split into multiple smaller and easier to read templates that 
+can eventually be reused in other States. For example, the following syntax is 
+not correct to manage the configuration of NTP and BGP simultaneously, using 
+two different templates and changing the device configuration through one 
+single commit:
+
+.. code-block:: yaml
+
+    manage_bgp_and_ntp:
+      netconfig.managed:
+        - template_name:
+            - salt://templates/bgp.jinja
+            - salt://templates/ntp.jinja
+        - context:
+            bpg: {{ pillar.bgp }}
+            ntp: {{ pillar.ntp }}
+
+Connection Re-establishment on Demand
+"""""""""""""""""""""""""""""""""""""
+
+Beginning with this release, any NAPALM command executed when
+running under a NAPALM Proxy Minion supports the ``force_reconnect``
+magic argument.
+
+Proxy Minions generally establish a connection with the remote network
+device at the time of the Minion startup and that connection is going to be
+used forever.
+
+If one would need execute a command on the device but connecting using
```

Comment:
`would need execute` --> `would need to execute`
and
`but connecting` --> `but is connecting`